src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -151,7 +151,8 @@ private ushort Significand
                 // says they should be equal, even if the signs differ.
                 return leftIsNegative || AreZero(left, right);
             }
-            return (left._value <= right._value) ^ leftIsNegative;
+
+            return (left._value == right._value) || (left._value <= right._value) ^ leftIsNegative;
```

Comment:
nit, more explicitly this should be (second half uses `<` not `<=`):
`return (left._value == right._value) || ((left._value < right._value) ^ leftIsNegative);`

It shouldn't matter since `==` was already covered by the first half, but its good to be consistent I think.

Reply:
ouch, I missed that. I'm not sure how we got this wrong initially if we were following Softfloat, but maybe I just proved that eyeballs are fallible..